src/ui/public/visualize/loader/pipeline_helpers/utilities.ts

```diff
@@ -98,3 +101,8 @@ export const getFormat = (mapping: any) => {
     return getFieldFormat(id, mapping.params);
   }
 };
+
+export const getTableAggs = (vis: Vis): any[] => {
+  const columns = tabifyGetColumns(vis.aggs.getResponseAggs(), !vis.isHierarchical());
```

Comment:
Leaving a note for myself that the use of `vis.isHierarchical()` here will conflict with #27154 -- I will update that PR accordingly once this is merged.